### PR TITLE
A various visual bug fixed on Country Finder 

### PR DESCRIPTION
**Pull request by webbyfox**
## To do (delete all that do not apply):

 - [x] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-669
 - [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added
 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by dahfool** [comment]
@webbyfox looks good, some the components need to be unit tested.

File: react-components/src/components/ProductFinder/CountryFinder.jsx [hunk 1]
```diff
@@ -93,20 +92,22 @@ export function CountryFinder(props) {
...
+      if (searchStr && country.name.toUpperCase().indexOf(searchStr) != 0) return ''
       return (
-        <li key={country.id}>
-          <button type="button" className="link m-r-s m-b-xs" data-country={country.name} data-id={country.id} data-region={country.region} onClick={selectCountry}>
-            {country.name}
-          </button>
-        </li>
+        <span className="c-1-5" key={index}>
+          <li>
+            <button type="button" className="link m-r-s m-b-xs" data-country={country.name} data-id={country.id} data-region={country.region} onClick={selectCountry}>
```
Comment (dahfool) [inline react-components/src/components/ProductFinder/CountryFinder.jsx:101]: @webbyfox Are you using `data-region`, data attributes not really needed in react ?

Comment (webbyfox) [reply react-components/src/components/ProductFinder/CountryFinder.jsx:101]: I am using to save in directory API for other stuff

**Review by dahfool — APPROVED** [review]

File: react-components/src/components/ProductFinder/CountryFinder.jsx [hunk 2]
```diff
@@ -93,20 +92,22 @@ export function CountryFinder(props) {
     })
   }
 
-  let regions = Object.keys(countryList || {}).map((region) => {
-    const countries = (countryList[region] || []).map((country) => {
-      if (searchStr && country.name.toUpperCase().indexOf(searchStr) !== 0) return ''
+  let regions = Object.keys(countryList || {}).map((region, index) => {
+    let countries = (countryList[region] || []).map((country, index) => {
```
Comment (bobmeredith) [inline react-components/src/components/ProductFinder/CountryFinder.jsx:96]: Why is this not const? I don't think it changes.

File: react-components/src/components/stylesheets/CountryChooser.scss [hunk 3]
```diff
@@ -108,15 +108,50 @@
...
+  width: fit-content;
+
+  &.icon {
+    font-weight: 900;
+    font-size: 40px;
+    margin-top:-15px;
+  }
+ 
+}
+h2.region-name{
```
Comment (bobmeredith) [inline react-components/src/components/stylesheets/CountryChooser.scss:133]: We tend to use helper functions for spacing like this. e.g. zero padding top and bottom = p-v-0,
margin bottom 0 = m-b-0, font-size would be included in body-l

**Review by bobmeredith — CHANGES_REQUESTED** [review]

**Pull request merged**